### PR TITLE
Update edgex-go's config test

### PR DESCRIPTION
**Pull request by MonicaisHer**
Github action test passed:  https://github.com/canonical/edgex-snap-testing/runs/6178703265?check_suite_focus=true

### CODE REVIEW

**Comment by MonicaisHer** [comment]
test passed: https://github.com/canonical/edgex-snap-testing/runs/6177208116?check_suite_focus=true
change GitHub action branch back to main.

**Review by farshidtz — COMMENTED** [review]
Looks good. The platform channel input is a useful addition.

Some minor comments.

File: test/suites/edgexfoundry/config_test.go [hunk 1]
```diff
@@ -7,34 +7,60 @@ import (
...
+	t.Cleanup(func() {
+		utils.SnapStop(t, platformSnap)
+	})
+	utils.SnapStart(t, platformSnap)
+
+	utils.SetGlobalConfig(t, platformSnap, snapAppName, deviceVirtualDefaultServicePort)
+}
+
+func TestMixedConfig(t *testing.T) {
+	utils.FullConfigTest = true
```
[the remark this answers is not in the document]
Comment (MonicaisHer) [reply test/suites/edgexfoundry/config_test.go:58]: Should I leave `utils.FullConfigTest = true` in config test, or move it to the global setup?

**Review by farshidtz — APPROVED** [review]
Thanks for the tests @MonicaisHer, the last of many for this sprint.

**Pull request merged**